recipes/default.rb

```diff
@@ -35,27 +35,34 @@
   notifies :reconfigure, 'chef_server_ingredient[supermarket]'
 end
 
-if node['supermarket_package']['package_source']
-  pkgname = ::File.basename(node['supermarket_package']['package_source'])
+if node['supermarket_omnibus']['package_url']
```

Comment:
i'm going to kill this branch, it was a quick hack for someone that was not happy that chef-server-ingredient wasn't working